### PR TITLE
Create categories

### LINKED ISSUE #6: Create categories
As a user, I want to be able to **create** and **see** a category.

### PR DESCRIPTION
**Pull request by donghua-shen**
Closes #6 

- [x] When there is no category on the categories page, display a relevant message.
- [x] Categories can only be created from the categories page.
- [x] If a user cancels when creating, the input will be discarded.
- [x] After the creation, the app will redirect to the categories page with the latest data.

### CODE REVIEW

File: Expired/Views/ProductCell.swift [hunk 1]
```diff
@@ -14,19 +14,32 @@ struct ProductCell: View {
...
         HStack {
             VStack(alignment: .leading) {
-                Text(product.title ?? "")
-                    .font(.headline)
-                    .padding(.bottom, 2)
+                HStack {
+                    Text(product.title ?? "")
+                        .font(.headline)
+                        .padding(.bottom, 2)
+                        .padding(.trailing, 6)
```
Comment (KarlReyes28) [inline Expired/Views/ProductCell.swift:21]:
```suggestion

```

Comment (KarlReyes28) [reply Expired/Views/ProductCell.swift:21]: I think this is not applicable anymore.

**Review by KarlReyes28 — COMMENTED** [review]
Done Reviewing! Thanks for the work!

**Review by sandeepsingh8 — APPROVED** [review]
Everything is working perfectly.

**Pull request merged**